Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorTransformComponentSelection.cpp

```diff
@@ -1898,9 +1898,9 @@ namespace AzToolsFramework
         {
             if (cursorEntityIdQuery.HasContainerAncestorEntityId())
             {
-                if (auto prefabFocusInterface = AZ::Interface<Prefab::PrefabFocusInterface>::Get())
+                if (auto prefabFocusPublicInterface = AZ::Interface<Prefab::PrefabFocusPublicInterface>::Get())
```

Comment:
question/nit: Most if not all interfaces are public? 😛 Maybe adding `public` here is redundant unless it really is adding something?

Reply:
True, all interfaces are public, which is kind of the point. Sometimes interfaces get crowded with functions that are meant for internal use inside the system alongside the ones that should be used by external systems and it gets confusing.

I've been trying to handle this in systems by either defining a regular, internal interface and a "Public" interface for external systems, or a regular, public interface and an "Internal" interface for internal calls. Not sure which naming convention is the best. If using "Internal" sounds better we can probably try to have a pass and streamline that setup, if not make it a recommendation in the style guide?

Reply:
Cool no problem, it's fine as is I just wanted to sanity check the reason why. Thanks!